apps/server/src/graphql/Task.ts

```diff
@@ -375,6 +392,52 @@ When the task is:
   })
 );
 
+builder.mutationField("makeTaskSubtask", (t) =>
+  t.prismaFieldWithInput({
+    type: "Task",
+    description: "Make a task a subtask of another task. Takes in the ID of the parent task and the ID of the subtask",
+    input: {
+      parentTaskId: t.input.id({ required: true, description: "The ID of the parent Task" }),
+      subTaskId: t.input.id({ required: true, description: "The ID of the subtask" }),
+    },
+    resolve: async (query, _, args) => {
+      const { parentTaskId, subTaskId } = args.input;
+
+      // Fetch the parent task and the subtask from the database
+      const parentTask = await prisma.task.findUnique({
+        where: { id: parseInt(parentTaskId) },
+      });
+      const subTask = await prisma.task.findUnique({
+        where: { id: parseInt(subTaskId) },
+      });
```

Comment:
nit: I think you can do this in 1 query. Not sure of the exact syntax, but should look something like this.
```suggestion
      // Fetch the parent task and the subtask from the database
      const [parentTaskId, subTaskId] = await prisma.task.find({
        where: { id: { in: [parseInt(parentTaskId), parseInt(subTaskId)] } },
      });
```